mdn/developer-portal
# 1420: Post-redesign minor model cleanup

This changeset contains three small changes as part of post-redesign cleanup:

* Drop 'show_header' option from `Homepage` model (Not relevant to the new design)
* Drop redundant `Topic.color` field and related constants and code
* Support showing a nickname for a person in their card, if configured, via the `Person.display_title` property

No nickname:
![Screenshot 2020-05-26 at 14 19 28](https://user-images.githubusercontent.com/101457/82907647-c9d2d080-9f5e-11ea-888a-ec9df2627154.png)

With nickname:

![Screenshot 2020-05-26 at 14 19 20](https://user-images.githubusercontent.com/101457/82907650-ca6b6700-9f5e-11ea-8714-b57ad49fde0e.png)


## How to test

- Code is deployed to `stage` and a new Person has been added with a nickname - confirm it shows as expected on both the overall listing page and the person-detail page. See [here](https://developer-portal.stage.mdn.mozit.cloud/communities/people/?page=4)
